L1Trigger/L1THGCal/src/be_algorithms/HGCalTriggerCellCalibration.cc

```diff
@@ -20,52 +19,34 @@ HGCalTriggerCellCalibration::HGCalTriggerCellCalibration(const edm::ParameterSet
 
 void HGCalTriggerCellCalibration::print(){
```

Comment:
Probably better to define a << operator like [here](https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_8_1_0_pre12/L1Trigger/L1THGCal/interface/fe_codecs/HGCalTriggerCellBestChoiceCodec.h#L8), this way you don't hardcode the output stream and can printout this using the edm log system (or anything you like)